WebApplication1/Models/Movie.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace WebApplication1.Models
+{
+    public class Movie
+    {
+        public int Id { get; set; }
+        public DateTime ReleaseDate { get; set; }
+        public Worker Director { get; set; }
```

Comment:
you need `int DirectorId` and make this to be `public virtual Worker Director`, the object that is referencing another table should be virtual unless it is a List or ICollection